### DISCUSSION TITLE
Why do the python aliases point to 3.11 and not the most recent 3.12?

### DISCUSSION THREAD

**Discussion by moritzreiter** [Homebrew/discussions#4981] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.25-52-ga04d8a3
ORIGIN: https://github.com/Homebrew/brew
HEAD: a04d8a3e12ee5522b4bb1aedbba4cd8ea2959b9d
Last commit: 24 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 7a834db0d03ce01727c110aa655e21064043cccf
Core tap last commit: 6 weeks ago
Core tap branch: master
Core tap JSON: 14 Dec 18:02 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.43.0 => /opt/homebrew/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.2-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: 15.1
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Is there a particular reason why the [`python`](https://github.com/Homebrew/homebrew-core/blob/master/Aliases/python), [`python3`](https://github.com/Homebrew/homebrew-core/blob/master/Aliases/python3) and [`python@3`](https://github.com/Homebrew/homebrew-core/blob/master/Aliases/python%403) aliases point to the `python@3.11` formula and not to the most recent version `python@3.12`?

**Comment by SMillerDev** [comment, marked as answer]
Because the pull request to do that is not merged yet

**Reply by moritzreiter** [reply]
Right, now I [found it](https://github.com/Homebrew/homebrew-core/pull/150390). Thanks for the hint!
